docs/products/clickhouse/howto/create-dictionary.md

```diff
@@ -0,0 +1,220 @@
+---
+title: Create dictionaries in Aiven for ClickHouse®
+sidebar_label: Create dictionary
+---
+
+Create dictionaries in Aiven for ClickHouse® to accelerate queries for better efficiency and performance.
+
+## Dictionaries in Aiven for ClickHouse
+
+A dictionary is a key -> attribute mapping useful for low latent lookup queries, when you
+often access reference lists to retrieve attributes from a key. Dictionary data resides
+fully in memory and is accessed using functions, which are faster than regular SQL queries.
+Dictionaries can be an efficient replacement for regular tables in your JOIN clauses.
+
+Aiven for ClickHouse supports
+[backup and restore](/docs/products/clickhouse/concepts/disaster-recovery#backup-and-restore)
+for dictionaries. Also, dictionaries in Aiven for ClickHouse are automatically replicated
+on all service nodes.
+
+Read more on dictionaries in the
+[upstream ClickHouse documentation](https://clickhouse.com/docs/en/sql-reference/dictionaries).
+
+## Prerequisites
+
+- Aiven for ClickHouse service created
+- SQL client installed
+- Dictionary source available
+- Credentials integration for remote ClickHouse, PostgreSQL®, or MySQL® if to be used as
+  sources
+
+## Limitations
+
+- TLS connections supported only (no non-TLS allowed)
+- In Aiven for ClickHouse, you run queries with the `avnadmin` user by default (as opposed
```

Comment:
This point is subtle and I think what is here is a bit ambiguous. It could be made clearer:


If no host is specified in a dictionary with ClickHouse source, the local host is assumed and the dictionary is filled with data from a query against the local ClickHouse.  For example:

```sql
-- users table
CREATE TABLE default.users
(
    id UInt64,
    username String,
    email String,
    country String
)
ENGINE = MergeTree()
ORDER BY id;

CREATE DICTIONARY default.users_dictionary
(
    id UInt64,
    username String,
    email String,
    country String
)
PRIMARY KEY id
SOURCE(CLICKHOUSE(DB 'default' TABLE 'users'))
LAYOUT(FLAT())
LIFETIME(100);
```

In Aiven for ClickHouse, to fill the dictionary the table `users` will be queried with the permissions of the `avnadmin` user - even if another user creates the dictionary.  In upstream ClickHouse, the same is true except the `default` user is used.